lessons/Algebra2/exploring-logarithmic-models/langs/en-us/pages/transforming-wealth-practice.adoc

```diff
@@ -0,0 +1,73 @@
+= Transforming Data (Practice)
+++++
+<style>
+table td:first-of-type p { font-weight: bold; }
+</style>
+++++
+
```

Comment:
Add link to starter file / CPO! (Flannery has said it enough times, it's stuck in my head too, lol.)